src/inject/dynamic-theme/css-rules.ts

```diff
@@ -1,29 +1,30 @@
 import {forEach} from '../../utils/array';
 import {isSafari} from '../../utils/platform';
 import {parseURL, getAbsoluteURL} from '../../utils/url';
-import {logWarn} from '../utils/log';
+import {logInfo, logWarn} from '../utils/log';
 
-export function iterateCSSRules(rules: CSSRuleList, iterate: (rule: CSSStyleRule) => void) {
+export function iterateCSSRules(rules: CSSRuleList, iterate: (rule: CSSStyleRule) => void, hasNonLoadedLinks: () => void = () => void 0) {
```

Comment:
`hasNonLoadedLinks` (`is`, `was`, `has`) means it will return boolean, but in your case it doesn't return anything. Try renaming it into what it actually does, e.g. `onMediaRuleError`.

Another minor problem I see is `() => void = () => void 0`. Maybe you could add `()` around each part? It is possible to replace `void 0` with `undefined` (it is immutable in modern browsers) or `Function.prototype`, or just strip off the assignment and check for the argument in the code.